terraform-consul-provider/overview.md

```diff
@@ -0,0 +1,128 @@
+## Overview of Workspace
+
+This is the workspace for the [**Register External Services with Terraform**](http://learn.hashicorp.com/consul/developer-discovery/terraform-consul-provider) guide.
+
+These services can be open by clicking on their respective tabs in the terminal.
+
+On initialization, this workspace should have ran `docker-compose up -d` which spins
+up the above services in the background. To view these instances, run `docker ps`{{execute}}
+
+It takes about a minute after you see the `Ready` message for the Consul datacenter
+to nominate a cluster leader and for you to continue to the next step.
+
+## Bootstrap Consul datacenter
+
+`docker exec -it consul-playground_consul-server-1_1 consul acl bootstrap`{{execute}}
+
+For more information about this command, please reference the guide.
+
```

Comment:
I think we can remove this bit since the guide covers it

Reply:
I think it's nice to click on the command to bootstrap Consul vs typing/copying and pasting it into the terminal

Reply:
👌 in that case lets just tell people where they should hop back to the guide. (move the current line 10 and have it describe the step they should jump back into)

Reply:
Not sure what you meant by current line 10... but I think I resolved it?

```
It takes about a minute after you see the `Ready` message for the Consul datacenter to nominate a cluster leader and for you to continue to the next [step](http://learn.hashicorp.com/consul/developer-discovery/terraform-consul-provider#bootstrap-consul-acls), bootstrapping your Consul ACLs.

## Bootstrap Consul ACLs

This corresponds to the [Bootstrap Consul ACLs](http://learn.hashicorp.com/consul/developer-discovery/terraform-consul-provider#bootstrap-consul-acls) section on the Learn guide. Once you run the following command, use the guide to complete the rest of this scenario.

`docker exec -it consul-playground_consul-server-1_1 consul acl bootstrap`{{execute}}
```